docs/advanced-features/self-signed-certificates.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+To use a self-signed certificate with the `quilt3` API or CLI,
+the certificate must first be added to your Python environment.
+
+## Mac and Linux
+
+1. Determine the location of your certificates:
+    ```python
+    >>> import certifi
+    >>> certifi.where()
+    '/path/to/site-packages/certifi/cacert.pem'
+    ```
+2. Copy your custom self-signed certificate and paste into the
+`cacert.pem` file.
+
+## Windows
+
+1. Press «Win + R» to open a new command prompt and type `certmgr`.
```

Comment:
Not sure if CP is the right app but the keystroke is an optional detail to actually opening the app.
```suggestion
1. Open Command Prompt («Win + R») to open a new command prompt and type `certmgr`.
```